content/spaces/quickstart/_index.md

```diff
@@ -439,7 +488,7 @@ With your kubeconfig pointed at the Kubernetes cluster where you installed the U
 
 ```yaml
 cat <<EOF | kubectl apply -f -
-apiVersion: spaces.upbound.io/v1alpha1
+apiVersion: spaces.upbound.io/v1beta1
 kind: ControlPlane
 metadata:
   name: ctp1
```

Comment:
I think we need to add a `metadata.namespace: default` to this YAML to make it work properly